gateway-messages/src/sp_to_mgs.rs

```diff
@@ -788,6 +1072,27 @@ impl fmt::Display for UpdateError {
             Self::MissingHandoffData => {
                 write!(f, "boot data not handed off to hubris kernel")
             }
+            Self::BlockOutOfOrder => {
+                write!(f, "bootloader blocks delivered out of order")
```

Comment:
Tiny nit - the `UpdateError` variant here could (at least in theory) come from something other than the bootloader - should we omit `"bootloader"` from the output?

(Same comment on `VersionNotSupported` below, where it might not come from RoT boot info?)

Reply:
you are correct